internal/pin/pin.go

```diff
@@ -91,7 +91,7 @@ func defaultOrchestrionToolGo() *dst.File {
 // PinOrchestrion applies or update the orchestrion pin file in the current
 // working directory, according to the supplied [Options].
 func PinOrchestrion(opts Options) error {
-	goMod, err := goenv.GOMOD()
+	goMod, err := goenv.GOMOD(".")
```

Comment:
```suggestion
	goMod, err := goenv.GOMOD("")
```

Previous behaviour was to use `""` and I am not knowledgeable enough to know if `"." ` is 100%  exactly the same thing on platforms like windows